dist/index.js

```diff
@@ -311,14 +311,22 @@ module.exports = _reactNative2.default.createClass({
    * @param  {number} index offset index
    */
   scrollTo: function scrollTo(index) {
```

Comment:
Can you also add the 'animated' parameter to the function signature, and document it above the function?